dask/array/tests/test_chunk.py

```diff
@@ -2,10 +2,12 @@
 
 pytest.importorskip("numpy")
 
+from operator import getitem as operator_getitem
+
```

Comment:
Nitpick: what you have here is totally correct, however using a non-standard name like `operator_getitem` might cause a future reader to have to look up what `operator_getitem` is. This isn't a huge deal, but is something to be aware of. Doing something like

```python
import operator

def test_getitem():
    ...
    y_op = operator.getitem(x, slice(120, 122))
    ...
```

instead will probably have slightly less cognitive overhead.